src/scanner/images/skopeo.ts

```diff
@@ -43,10 +45,10 @@ export async function pull(
   args.push({body: prefixRespository(image, SkopeoRepositoryType.ImageRegistry), sanitise: false});
   args.push({body: prefixRespository(destination, SkopeoRepositoryType.DockerArchive), sanitise: false});
 
-  await pullWithRetry(args);
+  await pullWithRetry(args, destination);
 }
 
-async function pullWithRetry(args: Array<processWrapper.IProcessArgument>): Promise<void> {
+async function pullWithRetry(args: Array<processWrapper.IProcessArgument>, destination: string): Promise<void> {
```

Comment:
clearly a hack because the destination is already included in the args list in a modified way.
I'm proposing this because we want to get rid of everything related to Skopeo anyway "soon"